lib/helm/chart2json.nix

```diff
@@ -24,37 +23,24 @@ with lib;
     noHooks ? false,
   }: let
     valuesJsonFile = builtins.toFile "${name}-values.json" (builtins.toJSON values);
-  in
-    stdenvNoCC.mkDerivation {
-      name = "${name}.json";
-      buildCommand = ''
-        # template helm file and write resources to yaml
-        helm template "${name}" \
+    resourcesYaml = runCommand "${name}.yaml" {nativeBuildInputs = [kubernetes-helm];} ''
```

Comment:
Creating separate derivations for <Helm> -> YAML -> JSON makes debugging the last step easier as `nix show-derivation` showes the single `yq` command which can be executed to reproduce the issue. It also contains the path to the `resources.yaml` file which can be inspected for issues.

Reply:
Should we add a comment inline saying as much? In case someone tries to simplify things by combining them down the line.

Reply:
Good point! Do you agree with splitting it up that way? The only downside I can think of is that we now have the `resources.yaml` in the nix store. Before we only had the raw helm chart, values.json and resources.json in the store.

Reply:
I don't have a strong opinion at this point. I'd combine them myself just for simplicity but happy to keep them separate if it helps others.